app/models/article.rb

```diff
@@ -123,12 +123,12 @@ class Article < ApplicationRecord
     .declare("l_org_vector tsvector; l_user_vector tsvector") do
     <<~SQL
       NEW.reading_list_document :=
-        to_tsvector('simple'::regconfig, unaccent(coalesce(NEW.body_markdown, ''))) ||
-        to_tsvector('simple'::regconfig, unaccent(coalesce(NEW.cached_tag_list, ''))) ||
-        to_tsvector('simple'::regconfig, unaccent(coalesce(NEW.cached_user_name, ''))) ||
-        to_tsvector('simple'::regconfig, unaccent(coalesce(NEW.cached_user_username, ''))) ||
-        to_tsvector('simple'::regconfig, unaccent(coalesce(NEW.title, ''))) ||
-        to_tsvector('simple'::regconfig,
+        setweight(to_tsvector('simple'::regconfig, unaccent(coalesce(NEW.body_markdown, ''))), 'C') ||
```

Comment:
Please feel free to ignore this but whenever I use `setweight` I order the clauses by the second argument so they start from `A`. This just makes it a tiny bit easier to grasp the column importance at a glance.

Reply:
done :)